vars/UpdateBuildFailureIssues.groovy

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+ /** Library to create, update an close GitHub issue across opensearch-project repositories for distribution build failures.
+ @param Map args = [:] args A map of the following parameters
+ @param args.failureMessages <required> - Failure message retrieved from buildFailureMessage() method.
+ @param args.passMessages <required> - Passing message retrieved from buildFailureMessage() method.
+ @param args.inputManifestPath <required> - Path to input manifest.
+ */
+void call(Map args = [:]) {
+    lib = library(identifier: 'jenkins@main', retriever: legacySCM(scm))
+    def failureMessages = args.failureMessages
+    def passMessages = args.passMessages
+    def yamlFile = readYaml(file: args.inputManifestPath)
```

Comment:
Change `yamlFile` to `inputManifestFile` or similar for better understanding.

Reply:
Done!